android/jni/tun2socks/tun2socks.c

```diff
@@ -128,7 +129,8 @@ struct {
     int set_signal;
     // ==== PSIPHON ====
     // ==== UPROXY ====
-    char *dns_server_address;
+    char *dns_resolver_address;
```

Comment:
Elsewhere in this `struct` `address` becomes `addr` - rename to `dns_resolver_addr` and `udp_relay_addr`?